src/main.go

```diff
@@ -155,7 +167,7 @@ func isTiDB(db *sql.DB) bool {
 }
 
 func (t *tester) addConnection(connName, hostName, userName, password, db string) {
-	mdb, err := OpenDBWithRetry("mysql", userName+":"+password+"@tcp("+hostName+":"+port+")/"+db+"?strict=true&time_zone=%27Asia%2FShanghai%27"+params)
+	mdb, err := OpenDBWithRetry("mysql", userName+":"+password+"@tcp("+hostName+":"+port+")/"+db+"?time_zone=%27Asia%2FShanghai%27"+params)
```

Comment:
what is the `strict` mean, why we need remove it

Reply:
The strict mode is an option in the driver that changes warnings to errors. However in newer versions of the driver this has been removed.

https://github.com/go-sql-driver/mysql/wiki/strict-mode